hpy/devel/src/runtime/argparse.c

```diff
@@ -27,9 +27,10 @@
  * -------
  *
  * O (object) [HPy *]
- *     Returns an existing handle. Not supported by HPyArg_ParseKeywords as
- *     retrieving an item from the keywords dictionary would require creating
- *     a new handled. Use O+ instead for this case.
+ *     Returns an handle. When using HPyArg_ParseKeywords, this might retrieve
+ *     a new handle from the keywords dictionary, so an HPyTracker pointer
+ *     must be passed and on successful return HPyTracker_Close must be called
+ *     to close any handles that were created.
  *
  * O+ (object) [HPy *]
```

Comment:
do we want to keep `O+` now that we have a nice way to deal with `O`?

I see that it might be convenient in some cases, but OTOH it makes it harder to read the code ("should I close this handle or not? Oh wait, let's look 50 lines above whether I used O+ or O")